aws/tfstate-backend/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+sed -i "s/backend/#backend/" main.tf
```

Comment:
this will change `backend` in two other places:
https://github.com/cloudposse/terraform-root-modules/blob/master/aws/tfstate-backend/main.tf#L54
https://github.com/cloudposse/terraform-root-modules/blob/master/aws/tfstate-backend/main.tf#L55

I did this before:
```
sed -i 's/backend          "s3"             {}/#backend          "s3"             {}/' main.tf
sed -i 's/#backend          "s3"             {}/backend          "s3"             {}/' main.tf```
```

but it's too fragile and could be broken if the file changes
We need to think of a better way.